adhocracy4/projects/models.py

```diff
@@ -135,36 +135,71 @@ def has_member(self, user):
     def has_moderator(self, user):
         return user in self.moderators.all()
 
-    @functional.cached_property
+    @property
     def other_projects(self):
         other_projects = self.organisation.project_set\
             .filter(is_draft=False, is_archived=False).exclude(slug=self.slug)
         return other_projects
 
-    @functional.cached_property
+    @property
     def is_private(self):
         return not self.is_public
 
-    @functional.cached_property
+    @property
+    def modules(self):
+        return self.module_set.all()
+
+    @property
+    def last_active_phase(self):
+        """
+        Return the last active phase.
+
+        The last active phase is defined as the phase that out of all past
+        and currently active phases started last.
+        """
+        return self.phases\
+            .past_and_active_phases()\
+            .last()
+
+    @property
     def last_active_module(self):
-        """Return the module of the currently active or last past phase."""
-        phase = self.active_phase or self.past_phases.first()
-        if phase:
-            return phase.module
+        """Return the module of the last active phase."""
+        last_active_phase = self.last_active_phase
+        if last_active_phase:
+            return last_active_phase.module
         return None
 
-    @functional.cached_property
+    @property
     def active_phase(self):
-        return self.phases\
-                   .active_phases()\
-                   .first()
+        """
+        Return the currently active phase.
+
+        The currently active phase is defined as the phase that out of all
+        currently active phases started last. This is analogous to the last
+        active phase.
+
+        Attention: This method is _deprecated_ as multiple phases may be
+        active at the same time.
```

Comment:
When writing this comment I was thinking about the following feedback:

> This changes our notion from one module can only be active to many modules can be active. But this restriction was actually a deliberate choice in order to simplify the modelling of real world processes/projects and because we don't have any concepts how to handle this ui and ux wise.

So in short my expectation was to fix this the other way round.